src/main/resources/templates/public_page.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Links</title>
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" rel="stylesheet">
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-C6RzsynM9kWDrMNeT87bh95OGNyZPhcTNXj1NW7RuBCsyN/o0jlpcV8Qyq46cDfL"
+            crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+
+    <style>
+        @media (min-width: 768px) {
+            .container {
+                width: 600px;
+            }
+        }
+
+        .container {
+            margin-top: 1rem;
+        }
+
+        .profile-picture {
+            width: 200px;
+            height: 200px;
+            border-radius: 50%;
+            display: block;
+            margin: 0 auto 2rem;
+        }
+
+        .profile-fullname {
+            text-align: center;
+            margin-bottom: 1rem;
+        }
+
+        .public-email {
+            text-align: center;
+            margin-bottom: 3rem;
+        }
+
+        .public-link {
+            text-align: center;
+            margin-bottom: 3rem;
+        }
+
+        .links {
+            display: flex;
+            flex-direction: column;
+            gap: 1rem;
+        }
+    </style>
+</head>
+<body>
+
+<nav th:replace="~{fragments/profile_navbar}"></nav>
+
+<div class="container">
+    <div th:if="${notFound != null}" class="alert alert-warning" role="alert">
+        <h4 class="alert-heading">404</h4>
+        <p>
+            Look like there is no data associated with this user.
+            Please, contact the person who shared this with you.
+        </p>
+    </div>
+
+    <div th:if="${notFound == null}">
+        <div class="public-link">
+            <h6>Copy link to share your profile:</h6>
+            <span th:text="${'http://localhost:8080/public/' + username}"></span>
```

Comment:
Why don't you make this a link? (`<a>`)

Reply:
No reason. I could have thought of it myself.